src/main/java/org/scijava/ui/UserInterface.java

```diff
@@ -163,6 +163,7 @@ default File chooseFile(final File file, final String style) {
 		// TODO use a utility class for style handling, e.g. StyleUtils.isStyle(style, ...)
 		if (style == null) title = "Choose a file";
 		else if (style.toLowerCase().contains(FileWidget.DIRECTORY_STYLE)) title = "Choose a directory";
+		else if (style.toLowerCase().contains(FileWidget.FILE_AND_DIRECTORY_STYLE )) title = "Choose a file or directory";
 		else if (style.toLowerCase().contains(FileWidget.OPEN_STYLE)) title = "Open";
 		else if (style.toLowerCase().contains(FileWidget.SAVE_STYLE)) title = "Save";
 		else title = "Choose a file";
```

Comment:
While at it, we might address the TODO comment here:

```suggestion
		if (style == null) title = "Choose a file";
		else if (WidgetStyle.isStyle(style, FileWidget.DIRECTORY_STYLE)) title = "Choose a directory";
		else if (WidgetStyle.isStyle(style, FileWidget.FILE_AND_DIRECTORY_STYLE )) title = "Choose a file or directory";
		else if (WidgetStyle.isStyle(style, FileWidget.OPEN_STYLE)) title = "Open";
		else if (WidgetStyle.isStyle(style, FileWidget.SAVE_STYLE)) title = "Save";
		else title = "Choose a file";
```

(needs the corresponding import for `org.scijava.widget.WidgetStyle`)